### PR TITLE
[VM Support] Modified code to identify the namespace of kvmservice instead of inputting the same

### PR DESCRIPTION
**Pull request by seswarrajan**

### CODE REVIEW

File: vm/vm.go [hunk 1]
```diff
@@ -15,15 +15,15 @@ import (
 )
 
 type VmOptions struct {
```
Comment (daemon1024) [inline vm/vm.go:17]:
```suggestion
type Options struct {
```

Unrelated to the PR changes, but can we handle this Lint Warning? Thank You.

Comment (seswarrajan) [reply vm/vm.go:17]: Sure @daemon1024 I will handle this.

**Review by nyrahul — CHANGES_REQUESTED** [review]
LGTM. Can you pls fix the lint issues introduced by new code change?

**Comment by daemon1024** [comment]
All of them are about lacking comments w.r.t. Exported Constants. In my opinion, Most exported consts are self-descriptive so we should configure the linter to ignore that. What do you think @nyrahul?

**Comment by nyrahul** [comment]
> All of them are about lacking comments w.r.t. Exported Constants. In my opinion, Most exported consts are self-descriptive so we should configure the linter to ignore that. What do you think @nyrahul?

I think linter is already been lenient it i.e., not failing the GH action. It is a good practice to add a comment for exported consts, funcs. Hence at least for new code it would be good to follow the practice.

**Review by nyrahul — APPROVED** [review]

**Pull request merged**